src/tds_fdw.c

```diff
@@ -1407,7 +1414,8 @@ void tdsGetColumnMetadata(ForeignScanState *node, TdsFdwOptionSet *option_set)
 						(errmsg("tds_fdw: Comparing retrieved column name to the following local column name: %s", local_name)
 						));
 
-					if (strncmp(local_name, column->name, NAMEDATALEN) == 0)
+					//! \PPV local_name always in lowcase. But column->name's register defined in sql server. so you should ignore case
+					if (strnicmp(local_name, column->name, NAMEDATALEN) == 0)
```

Comment:
It seems this is crashing tds_fdw on loading: https://jenkins.juliogonzalez.es/job/tds_fdw-build-pr/44/DISTRO=ubuntu14.04,PG_VER=9.2,label=docker/console

Could it be that ```strnicmp``` is not part of the standard library and MS stuff only?